leptonai/cli/photon.py

```diff
@@ -563,6 +596,10 @@ def run(
         # parse environment variables and secrets
         deployment_name = _find_deployment_name_or_die(conn, name, id, deployment_name)
 
+        if include_workspace_token:
+            console.print("Including the workspace token for the photon execution.")
+            _add_workspace_token_secret_var_if_not_existing(conn)
```

Comment:
do users need to explicitly specify `-s LEPTON_WORKSPACE_TOKEN` when launching a deployment?